projects/components/src/toggle-button-group/toggle-button-group.component.ts

```diff
@@ -39,6 +39,9 @@ export class ToggleButtonGroupComponent implements OnChanges, AfterViewInit {
   @Input()
   public disableInitialSelection?: boolean;
 
+  @Input()
+  public justifyContent?: string;
```

Comment:
Should we follow an enum based mode approach, instead of passing styles directly as an input?

Reply:
I've already made those changes. will update PR soon :)

Reply:
@skjindal93 done